_docs_use-cases/use-case-transport-create-own-relationshiptemplate.md

```diff
@@ -61,3 +62,4 @@ A [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relat
 
 - The `content` is malformed.
 - `expiresAt` lies in the past.
+- In case of password protection of the RelationshipTemplate, a `passwordProtection.password` that does not consist of 4 to 16 digits was specified, but the value of `passwordProtection.passwordIsPin` was nevertheless set to `true`.
```

Comment:
```suggestion
- In case of a password protected RelationshipTemplate, a `passwordProtection.password` that does not consist of 4 to 16 digits was specified, but the value of `passwordProtection.passwordIsPin` was nevertheless set to `true`.
```
or
```suggestion
- In case of a password protection of the RelationshipTemplate, a `passwordProtection.password` that does not consist of 4 to 16 digits was specified, but the value of `passwordProtection.passwordIsPin` was nevertheless set to `true`.
```

Same in other places.

Reply:
I know the wording I used to be acceptable.